addons/crm/tests/test_crm_lead.py

```diff
@@ -676,7 +694,50 @@ def test_message_get_suggested_recipients(self):
             'lang_id': lang_en.id,
         })
         data = lead2._message_get_suggested_recipients()[lead2.id]
-        self.assertEqual(data, [(False, self.test_email, "en_US", 'Customer Email')])
+        self.assertEqual(len(data), 1)
+        self.assertEqual(data[0][:-1], (False, self.test_email, "en_US", 'Customer Email'))
```

Comment:
Mmmh, it is a bit obscure :D why not being a bit more explicit and using variables, like ``partner_id, partner_email, lang, reason, create_vals = data[0]`` and then do explicit checks ?

Reply:
Yes, indeed, I was trying to be as compact as the previous version but the new format don’t allow to do something very clean. I have added the explicit checks instead.